addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -45,10 +45,10 @@
                         <i class="fa fa-share-alt"/>
                         <span class="d-none d-md-inline-block ms-2">Share</span>
                     </a>
-                    <a class="d-flex align-items-center px-3 o_wslides_fs_exit_fullscreen" t-attf-href="/slides/slide/#{slug(slide)}">
+                    <a class="d-flex align-items-center px-3 o_wslides_fs_exit_fullscreen" t-attf-href="/slides/slide/#{slug(slide)}" title="Exit Fullscreen">
```

Comment:
A title has been added instead of an `aria-label` in order to be consistent with the "Share" anchor (see the anchor before this one)